coss/opensource_clubs/models.py

```diff
@@ -117,9 +148,19 @@ class EntityPageTag(TaggedItemBase):
 
 class EntityDetailPage(Page):
     """Actual page for the Open Source clubs."""
+
+    entity_title = models.CharField(max_length=128, default='')
     description = fields.RichTextField(blank=True)
     featured = models.BooleanField(default=False)
+    is_flagged = models.BooleanField(default=False)
+    club_name = models.CharField(max_length=128, default='')
+    discourse_link = models.URLField(verbose_name='Discourse Link', blank=True, default='')
```

Comment:
Shouldn't we use the Discourse snippet here?

Reply:
I thought that it's faster to proceed with this for now.  This whole page will be refactored. Regarding the snippet, it would be great if we had a method to construct the url there. I will open a ticket for that and we can swap it out later when all the major components are done.